iTwin/appui
# Allow `WidgetDef.show()` to work with popouts

## Changes
Changed widgets show() function to work with popouts as well.
This behavior ended up greatly depending on the browser used and is not guaranteed.
Just calling window.focus() worked for all browsers except Safari (but might differ depending on the user settings).
Safari has security measures that prevent you from changing focus to a different tab programmatically. Apple support docs to issue: https://discussions.apple.com/thread/251676767?sortBy=best . 

## Testing

- Test Chrome, Edge, Safari, Electron, and Firefox and evaluate the behavior. There is a widget in the WidgetApi frontstage that enables you to call show on any widget, which makes for easy testing.

I.e.:
- Start standalone test-app
- Open <http://localhost:3000/?frontstage=appui-test-providers:WidgetApi>
- Popout `WL-A` widget
- Open `Layout Controls` widget
- In the `Widget id` input select `WL-A`
- Click `Show` button
- Observe how the window of the popout widget is brought to front (if supported by the browser)
